spec/unit/unmanaged_files_inspector_spec.rb

```diff
@@ -131,7 +131,7 @@ def expect_cat_mounts(system, add_files)
       expect(system).to receive(:read_file).and_return(ret_val)
     end
 
-    def expect_find_commands(system,add_files)
+    def expect_find_commands(system,add_files, filtered_paths)
```

Comment:
Assignment Branch Condition size for expect_find_commands is too high. [21.47/15]<br>Space missing after comma.